mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -47,6 +49,39 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
   private var currMax: Array[Double] = _
   private var currMin: Array[Double] = _
 
+  private var configure: mutable.Map[String, Boolean] = mutable.Map("mean" -> false, "max" -> false,
+    "min" -> false, "variance" -> false, "normL1" -> false, "normL2" -> false) 
+
```

Comment:
since in the tight loop, you are checking the condition all the time, let's not use map, but use primitive type for now.